frontend/src/hooks/useMount.ts

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
Oh interesting! I've been wondering what the best way of doing the `componentDidMount` equivalent is since `useEffect` seemed to combine `componentDidMount`, `componentDidUpdate`, and `componentWillUnmount`

Reply:
I'm guessing just passing an empty dependency array to `useEffect` wasn't desired behaviour because you also want it to run on unmount?

Reply:
Just using `useEffect(..., [])` would be the same thing, but it'll complain about missing dependencies.  This way it's obvious what we're doing and reduces the extra lines of code.

Reply:
Ah gothcya! Makes sense